### ISSUE TITLE
Move AWS policies to Tower user docs

### ISSUE THREAD

**Issue by bentsherman** [seqeralabs/nf-tower-aws#5]
For the documentation session today I am thinking about incorporating this repo into the tower user docs like I did with nf-tower-k8s. Looks like the instructions are already there, just need to move the policy files. @pditommaso let me know if you have any objections.

**Comment by pditommaso** [comment]
Yes, provide they will leave in their own page. This is needed because they should be linked from the app. See below

<img width="1068" alt="Screenshot 2022-04-08 at 15 17 25" src="https://user-images.githubusercontent.com/816968/162443498-6469bdca-b30e-4e71-8ec1-5250aca92ff4.png">

**Comment by bentsherman** [comment]
That link goes to the Tower Forge README: https://github.com/seqeralabs/nf-tower-aws/blob/master/forge/README.md

We could instead link to the AWS Batch Forge section in the Tower docs: https://help.tower.nf/22.1/compute-envs/aws-batch/

Or just say "go to the Tower user docs" since the link is tied to a specific version.

**Comment by bentsherman** [comment]
The Tower docs have been updated, and this repo can now be archived. I will submit issues to update nf-tower-cloud and seqera.io/docs.

**Comment by bebosudo** [comment]
Tracked in https://seqera.atlassian.net/browse/DEVOPS-954
